processor/deltatocumulativeprocessor/internal/data/expo/merge.go

```diff
@@ -26,10 +26,10 @@ func Merge(arel, brel Buckets) {
 	size := up - lo
 
 	counts := pcommon.NewUInt64Slice()
```

Comment:
[**For reviewers**] There is also some opportunity to reduce allocations a bit more if `arel` is already at the required capacity and the lower bound of the final result is the `arel`'s lower bound but I haven't done that yet since that might require some discussion over readability vs performance.